takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/AbstractCompiler.java

```diff
@@ -197,4 +207,6 @@ public void skipCompile() {
     context.markUptodateExecution();
   }
 
+  public abstract Set<String> getReferencedClasspathEntries();
```

Comment:
Should return `Set<File>` for symmetry with `#setClasspath(List<File>, File, Set<File>)`